application/assets/config/chromeExtension.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": "01-2020",
+  "manifestUrl": "https://raw.githubusercontent.com/hmrc/play-frontend-govuk-examples/master/src/test/resources/manifest.json",
+  "examplesRootUrl": "https://raw.githubusercontent.com/hmrc/play-frontend-govuk-examples/master/src/test/",
+  "hmrcComponentPathPrefix": "hmrc/",
```

Comment:
I think we should use `vendor` instead of `hmrc` because I'm discussing this being used by other departments.